### ISSUE TITLE
[feature request] Opt-in Setting for no instant search/call on Ctrl+d

### ISSUE THREAD

**Issue by Medivha** [Sidekick-Poe/Sidekick#289]
Would be nice to have an option in the settings to opt-in so when you Ctrl+d that it opens the window but not instantly searches for the item on the site.
It happens a lot that you have to checkmark another mod or 2 and search again, so you already used 2 trade API requests. 
not only do you drain the API unnecessary, you also are faster rate limited because of that.

I understand that it's probably annoying for the normal user, that's why I would like it as an opt-in setting. ❤️

**Comment by leMicin** [comment]
We don't auto search for normal/magic/rare items, I think it may make sense we don't auto search for uniques as well. Do you have other item types / rarities that this option would be useful for?

**Comment by Medivha** [comment]
Technically it would apply to currency as well if you want to check bulk search and yes i mainly noticed it on uniques and didn't check all types that are searchable ;)

**Comment by leMicin** [comment]
Yeah, it might be better to just always wait for a search click. It could simplify settings as well as we could remove some.
![image](https://github.com/user-attachments/assets/1b7b5e39-82da-4724-933b-9e70724905ec)
